src/absil/il.fs

```diff
@@ -13,18 +13,12 @@ open Microsoft.FSharp.Compiler.AbstractIL
 open Microsoft.FSharp.Compiler.AbstractIL.Internal
 open Microsoft.FSharp.Compiler.AbstractIL.Internal.Library
 open Microsoft.FSharp.Compiler.AbstractIL.Diagnostics
-open System
 open System.Collections
 open System.Collections.Generic
 open System.Collections.Concurrent
  
 let logging = false 
 
-let runningOnWindows =
```

Comment:
System.Environment.OSVersion is unsupported on dotnetcore, that's why I used the ms/visualfsharp version here.